src/sidebar/components/MarkdownView.tsx

```diff
@@ -22,14 +44,49 @@ export default function MarkdownView({
   markdown,
   classes,
   style,
-  mentions,
```

Comment:
Workaround for coverage issue: https://github.com/istanbuljs/istanbuljs/issues/812#issuecomment-2656077301